tests/test_cmd2.py

```diff
@@ -1825,3 +1825,74 @@ def test_onecmd_raw_str_quit(base_app):
     assert stop
     assert out == ''
 
+
+@pytest.fixture(scope="session")
```

Comment:
I added this fixture to help ensure that the history file always gets deleted no matter what